docs/project_defn.rst

```diff
@@ -222,8 +222,39 @@ on the `AWS Batch <https://aws.amazon.com/batch/>`_ service.
 
 GCP Configuration
 ~~~~~~~~~~~~~~~~~
-TODO: Add GCP configuration details
+The top-level ``gcp`` key is used to specify options for running the batch job
+on the `GCP Batch <https://cloud.google.com/batch>`_ service.
 
+.. note::
+    When BuildStockBatch is run on GCP, it will only save results to GCP Cloud Storage (using the
+    ``gcs`` configuration below); i.e., it currently cannot save to AWS S3 and Athena. Likewise,
+    buildstock run locally, on Eagle, or on AWS cannot save to GCP.
+
+*  ``job_identifier``: A unique string that starts with an alphabetical character,
+   is up to 10 characters long, and only has letters, numbers or underscore.
+   This is used to name all the GCP service objects to be created and
+   differentiate it from other jobs.
+*  ``project``: The GCP Project ID in which the batch will be run and of the Artifact Registry
+   (where Docker images are stored).
+*  ``gcs``: Configuration for project data storage on GCP Cloud Storage.
+
+    *  ``bucket``: The Cloud Storage bucket this project will use for simulation output and
+       processed data storage.
+    *  ``prefix``: The Cloud Storage prefix at which the data will be stored.
+
+*  ``region``: The GCP region in which the batch will be run and of the Artifact Registry.
+*  ``use_spot``: true or false. Defaults to false if missing. This tells the project
+   to use `Spot VMs <https://cloud.google.com/spot-vms>`_ for data
+   simulations, which can reduce costs by up to 91%.
+*  ``batch_array_size``: Number of concurrent simulations to run. Max: 10000.
+*  ``notifications_email``: Email to notify you of simulation completion.
```

Comment:
This doesn't work yet, so let's leave a TODO to update the email details when it does. I'm not sure that the part about having to accept emails will be the same as on AWS.

Reply:
K, I'm just going to remove it from this spot (so documentation isn't out of sync with the state of code) and from the config spec, but I've put todo there.